tests/experimental/datasources/test_pandas_datasource.py

```diff
@@ -213,7 +223,7 @@ def test_pandas_sorter(pandas_datasource, csv_path, order_by):
     @dataclass(frozen=True)
     class TimeRange:
         key: str
-        range: List[int]
+        range: List[str]
```

Comment:
This change eliminates `PyCharm` warning.

Reply:
Is this correct though?
@billdirks

Reply:
@Kilo59 Yes it is correct, because @billdirks identified this during our code walkthrough.